library/Backends/llvm/src/Juvix/Backends/LLVM/Codegen/Block.hs

```diff
@@ -694,7 +699,7 @@ printCString str args = do
 defineMalloc :: External m => m ()
 defineMalloc = do
   let name = "malloc"
-  op <- external (Types.pointerOf Type.i8) name [(Types.size_t, "size")]
+  op <- external (Types.pointerOf Type.i8) name [(Type.i32, "size")]
```

Comment:
What made it necessary to move away from `size_t`?

Reply:
the LLVM `size_of` gives us back a `i32` and I didn't want to deal with bit casting it to the right type, so I just moved these to i32's for now

Reply:
Namely because `sizeof` returns an i32, and I didn't feel like trying to bitcast it away, so I just left it as is, and demoted malloc to an i32. we can change it back and note of the hack